### ISSUE TITLE
Feature Request - show version badges to installed packages

### ISSUE THREAD

**Issue by berecont** [contao/contao-manager#823]
Feature Request

Is it possible to display the new version badges in the installed packages as well?
This would have the advantage of giving a first impression of whether the Contao version can be upgraded.

**Comment by aschempp** [comment]
you can see that information in the details popup already. I think showing this in the list of _installed_ packages could be misleading, because the version you currently use does not match the shown compatibility.

**Comment by zoglo** [comment]
Wouldn't be "too misleading" (take it with a grain of salt), it's more about "where" to show it.

Been thinking about adding it when I did the PR in the package-list, however it's more about way too many information that are visible at once (mind that there is also the responsive view) 🤔

**Comment by fritzmg** [comment]
> because the version you currently use does not match the shown compatibility.

When would that be the case?

**Comment by zoglo** [comment]
> When would that be the case?

E.g. using in `terminal42/contao-leads` in version 1.x for C4.13 but 3.x being the one for Contao 5.
The version that you have in your composer.json wouldn't be compatible with C5.

**Comment by berecont** [comment]
I think there is a misunderstanding. I meant that the possible compatibility of the versions is shown here - regardless of the version you have installed. 

In other words, just as it is in the general overview. The installed version is not queried here, but the latest package and the highest compatibility of the Contao version is displayed.

**Comment by fritzmg** [comment]
> E.g. using in `terminal42/contao-leads` in version 1.x for C4.13 but 3.x being the one for Contao 5.
> The version that you have in your composer.json wouldn't be compatible with C5.

But the extension in general is compatible with Contao 5 - that's the point. That's what this feature request is about. If you want to update from Contao 4.13 to Contao 5 you'd want to know at a glance whether all of your extensions are compatible  with Contao 5. If you have 20 extensions installed you'd not want to klick into the detail view 20 times just to check if one of them does not have the badge for Contao 5.

**Comment by zoglo** [comment]
If someone has a good design proposal, I wouldn't mind implementing it.

It just feels very stuffed right now with all the information

**Comment by berecont** [comment]
![cm-feature-request-823](https://github.com/user-attachments/assets/bfeb7855-ae5a-4353-82e7-aacd48858dc6)
just put `<ul>` before the `<h1>`  
same as in the general view

**Comment by zoglo** [comment]
I agree with the other versions but this is very stuffed and completely out of place
![image](https://github.com/user-attachments/assets/868ff57f-6c48-4383-b19f-187c17e495d3)

**Comment by fritzmg** [comment]
Looks fine to me. If anything, the download, license and like information could be removed. Surely the Contao Version compatibility is more useful in the installed packages screen than those.

**Comment by fritzmg** [comment]
imho it does not need to re removed. I assume it's there so it is consistent with the package search. The same should apply to the version badges. But if something has to removed, it should not be the version badges, as that is arguably more valuable information than the amount of downloads, likes or the license information.

**Comment by aschempp** [comment]
I would probably place it beside the stats. I don't really like it though, but I don't know why. I agree the stats are not important either, maybe we should revise the package list in general. But not as a quick fix 😏
